resources/qml/Preferences/Materials/MaterialsView.qml

```diff
@@ -222,7 +222,22 @@ Item
                     id: colorDialog
                     title: catalog.i18nc("@title", "Material color picker")
                     color: properties.color_code
-                    onAccepted: base.setMetaDataEntry("color_code", properties.color_code, color)
+//                    swatchColors: ["#2161AF", "#57AFB2", "#F7B32D", "#E33D4A", "#C088AD"]
+                    onAccepted: {
+                        base.setMetaDataEntry("color_code", properties.color_code, color);
+                        console.log("color_code");
```

Comment:
```suggestion
```

Reply:
Oops forgot to remove some debugging code